src/data.rs

```diff
@@ -58,6 +58,7 @@ pub struct CollectorParams {
     pub data_dir: String,
     pub run_name: String,
     pub profile: HashMap<String, String>,
+    pub tmp_dir: String,
```

Comment:
Let's use PathBuf for paths. We should change the other members of the CollectorParams from String to PathBuf.